packages/core/core/src/public/Bundle.js

```diff
@@ -282,7 +283,26 @@ export class PackagedBundle extends NamedBundle implements IPackagedBundle {
     return packagedBundle;
   }
 
+  static getWithInfo(
```

Comment:
I couldn't get Flow to let me add an additional argument to the existing `get` function...